internal/sdkv2provider/schema_cloudflare_pages_project.go

```diff
@@ -130,21 +131,25 @@ func resourceCloudflarePagesProjectSchema() map[string]*schema.Schema {
 				Type:        schema.TypeMap,
 				Description: "KV namespaces used for Pages Functions.",
 				Optional:    true,
+				Computed:    true,
 			},
 			"durable_object_namespaces": {
 				Type:        schema.TypeMap,
 				Description: "Durable Object namespaces used for Pages Functions.",
 				Optional:    true,
+				Computed:    true,
 			},
 			"d1_databases": {
 				Type:        schema.TypeMap,
 				Description: "D1 Databases used for Pages Functions.",
 				Optional:    true,
+				Computed:    true,
```

Comment:
why are these bindings being changed to computed?

Reply:
They are being sent from the backend.
From a debug log
```
The following problems may be the cause of any confusing errors from downstream operations:
      - .deployment_configs[0].preview[0].d1_databases: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      - .deployment_configs[0].preview[0].durable_object_namespaces: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      - .deployment_configs[0].preview[0].kv_namespaces: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      - .deployment_configs[0].preview[0].usage_model: planned value cty.StringVal("bundled") for a non-computed attribute
      - .deployment_configs[0].preview[0].always_use_latest_compatibility_date: planned value cty.False for a non-computed attribute
      - .deployment_configs[0].preview[0].environment_variables: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      - .deployment_configs[0].preview[0].fail_open: planned value cty.False for a non-computed attribute
      - .deployment_configs[0].preview[0].r2_buckets: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      - .deployment_configs[0].preview[0].secrets: sensitive planned value for a non-computed attribute
      - .deployment_configs[0].production[0].always_use_latest_compatibility_date: planned value cty.False for a non-computed attribute
      - .deployment_configs[0].production[0].d1_databases: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      - .deployment_configs[0].production[0].usage_model: planned value cty.StringVal("bundled") for a non-computed attribute
      - .deployment_configs[0].production[0].durable_object_namespaces: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      - .deployment_configs[0].production[0].environment_variables: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      - .deployment_configs[0].production[0].fail_open: planned value cty.False for a non-computed attribute
      - .deployment_configs[0].production[0].kv_namespaces: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      - .deployment_configs[0].production[0].r2_buckets: planned value cty.MapValEmpty(cty.String) for a non-computed attribute
      -
```
Previously, I recommended using empty deployment configs, but this improves user experience as they won't need to have them.

Reply:
this isn't really what `Computed` on a schema attribute is meant to be used for. `Computed` attributes are ones where we don't know the value at runtime but by the end of the operation we should. examples are if we send a payload to generate a new resource but we don't know what the `id` field is since the remote will be generating it. this looks to be more like that we are setting empty schema attributes or conditionally setting them when we should really always be providing a value (even `null` where it makes sense).

Reply:
Removed the attempted fix for this